pages/components.html

```diff
@@ -0,0 +1,451 @@
+            style="
+              width: 100%;
+              height: 577px;
+              border: 0;
+              transform: scale(1);
+              overflow: hidden;
+            "
+            sandbox="allow-scripts allow-same-origin"
+          >
+          </iframe>
+        </section>
+        <section id="doc--description--badges" class="doc-description--badges">
+          <span class="badge primary--badge">1</span>
+          <span class="badge secondary--badge">new</span>
+          <span class="badge alternate--badge">new</span>
+          <span class="badge dark--badge">new</span>
+          <iframe
+            src="https://carbon.now.sh/embed?bg=rgba%28255%2C255%2C255%2C1%29&t=night-owl&wt=none&l=htmlmixed&ds=false&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Cspan%2520class%253D%2522badge%2520primary--badge%2522%253E1%253C%252Fspan%253E%250A%253Cspan%2520class%253D%2522badge%2520secondary--badge%2522%253Enew%253C%252Fspan%253E%250A%253Cspan%2520class%253D%2522badge%2520alternate--badge%2522%253Enew%253C%252Fspan%253E%250A%253Cspan%2520class%253D%2522badge%2520dark--badge%2522%253Enew%253C%252Fspan%253E"
+            style="
+              width: 100%;
+              height: 260px;
+              border: 0;
+              transform: scale(1);
+              overflow: hidden;
+            "
+            sandbox="allow-scripts allow-same-origin"
+          >
+          </iframe>
+        </section>
+
+        <section
+          id="doc--description--buttons"
+          class="doc--description--buttons"
+        >
+          <button class="button primary--button">primary</button>

```

Comment:
Try giving 
cursor: pointer

to all buttons. It give an important visual feedback to the user that it is interactive